ui/src/attestation/contract.ts

```diff
@@ -53,22 +63,34 @@ export class ClaimsContract {
   // Fetches the identity claimed by the given Ethereum address.
   // Returns `undefined` if currently there's no valid claim.
   async getClaimed(address: string): Promise<Uint8Array | undefined> {
+    const normalisedAddress = address.toLowerCase();
+    const cached: ClaimsContractCacheEntry | undefined =
```

Comment:
Any reason why we don’t use `ui/src/memoizeLru` to separate the caching responsibility?

Reply:
I couldn't figure out how to make it work within a class.

Reply:
Ah, of course.